src/shared/components/Tabs/TabList.tsx

```diff
@@ -15,15 +18,20 @@ type TabListProps = {
   onSelected: (activeKey: string) => void;
   defaultActiveId?: string;
   position?: 'left' | 'right' | 'top' | 'bottom' | undefined;
+  onEditClick? : ( id : string) => void 
 };
 
+
+
 const TabList: React.FunctionComponent<TabListProps> = ({
   items,
   onSelected,
   defaultActiveId,
   position = 'left',
   children,
+  onEditClick
 }) => {
+  const editClick = onEditClick?  onEditClick :  ( id: string ) => {};
```

Comment:
A cheap way to accomplish default prop. I may remove this once we have workspace/dashboard edit logic developed and onEditClick becomes a mandotory prop.